solutions/community-deploy/cloud/AWS/README.md

```diff
@@ -4,7 +4,15 @@ This document discusses the procedure to compile the CORTX Stack and deploy on A
 
 **Prerequisites:**
 
- - During tools installation, you will be prompted to enter your AWS Access and Secret key so ensure that you have an AWS account with Secret Key and Access Key. For more details, refer [AWS CLI Configuration](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-config).
+ - Install Prerequisite tools by executing following commands:
+ ```
```

Comment:
Can we remove this as `./tool_setup.sh` script installs AWS CLI and Terraform as per - https://github.com/Seagate/cortx-re/blob/main/solutions/community-deploy/cloud/AWS/README.md#install-and-setup-terraform-and-aws-cli

Reply:
fixed